src/controlCenter/controlCenter.vala

```diff
@@ -207,6 +205,53 @@ namespace SwayNotificationCenter {
             });
         }
 
+        /**
+         * Sets the notification window width and height to the user configured
+         * values, taking into account the monitor width and height.
+         * If the configured values are above the monitor's width/height, they
+         * will be clipped to the monitor dimensions.
+         */
+        public void on_map () {
+            // "Constants"
+            bool anchorTop = ConfigModel.instance.positionY == PositionY.TOP;
+            bool fitToScreen = ConfigModel.instance.fit_to_screen;
+            bool anchorLeft = ConfigModel.instance.positionX == PositionX.LEFT;
+            bool anchorRight = ConfigModel.instance.positionX == PositionX.RIGHT;
```

Comment:
This is messy and ugly.
Maybe I should refactor the logic to set the anchors to a distinctive function?

Reply:
Hmmm. Clamping the width and height values seems a little out of reach atm so it would be better to just create a new PR with that enhancement and let this PR focus on setting the width and height

Reply:
Done.